alpakaConfig.cmake

```diff
@@ -381,12 +381,12 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
             ELSE()
                 IF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
                     IF(CUDA_VERSION VERSION_EQUAL 8.0)
-                        IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 5.0)
-                            MESSAGE(FATAL_ERROR "NVCC 8.0 does not support GCC 6+ and fails compiling the std::tuple implementation in GCC 5+. Please use GCC 4.9!")
+                        IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 5.4)
+                            MESSAGE(FATAL_ERROR "NVCC 8.0 does not support GCC 6+ and fails compiling the std::tuple implementation in GCC 5.4+. Please use GCC 4.9 - 5.3!")
                         ENDIF()
                     ELSEIF((CUDA_VERSION VERSION_EQUAL 9.0) OR (CUDA_VERSION VERSION_EQUAL 9.1))
                         IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 6.0)
-                            MESSAGE(FATAL_ERROR "NVCC 9.0-9.1 do not support GCC 7+ and fail compiling the std::tuple implementation in GCC 6+. Please use GCC 4.9 or 5!")
+                            MESSAGE(FATAL_ERROR "NVCC 9.0 - 9.1 do not support GCC 7+ and fail compiling the std::tuple implementation in GCC 6+. Please use GCC 4.9 - 5.5!")
```

Comment:
I would not mention 5.5 here explicitly because ther might be a 5.6 patch release and this will most probably work as well and we will not even find out that there was a new release. `4.9` also does not mention the patch release. gcc changed its versioning scheme between 4.9 and 5 so `4.8`, `4.9`, `5`, `6` should be the way to mention those releases. Only in the rare case that a patch release breaks something, I would explicitly mention it.

Reply:
My thinking was the following:
- 5.6 would still be allowed without a CMake error, but we know *at the time of release* the last tested version and should recommend that in warnings/errors to avoid further, cascading tries
- 5.6 would *not* be a patch release but a *minor*, which can potentially break things (see 5.3 vs 5.4 in NVCC 8.0)